### PR TITLE
WD-10146 Build new Confidential AI page

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Go to https://ubuntu-com-13812.demos.haus/confidential-computing/ai and check against [copy doc](https://docs.google.com/document/d/1TZ_jK3ReetAU2qA_YxfZL8i4j_lFidcF6F2Xup8R8Nc/edit)
- Also check against design review in the screenshot below

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10145

## Screenshots ([design review](https://www.figma.com/file/l3WhG4JN4JcmVjC617rhkY/Ubuntu.com%2Fconfidential-ai?type=design&mode=design&t=u4X80EanX4RiYRkE-0))

Desktop

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by camillet16** [comment]
Hi Carlos here's some feedback:

Screenshot 2024-04-26 at 3 23 42 PM
new-image: https://assets.ubuntu.com/v1/3ff12e71-Image.png

Screenshot 2024-04-26 at 3 23 26 PM
Screenshot 2024-04-26 at 3 24 26 PM
Screenshot 2024-04-26 at 3 24 23 PM

**Comment by juanruitina** [comment]
- [ ] Please implement the navigation as specified in the design (this is now a bubble)
- [ ] I think section HRs are missing the `p-rule` class, spacing is a bit off
- [ ] The diagram under "Confidential AI with Ubuntu and Nvidia H100 GPUs" should have a good alt text, I've asked Ijlal in the [copydoc](https://docs.google.com/document/d/1TZ_jK3ReetAU2qA_YxfZL8i4j_lFidcF6F2Xup8R8Nc/edit?disco=AAABMlFnoY0).
- [ ] List items under "Learn more about confidential computing" should not be checklists
- [ ] Update the copy doc metatag

The three subsections under "How confidential AI works" look a bit weird on medium viewports, @lyubomir-popov should we address this?

**Comment by carkod** [comment]
@camillet16 can I get another review?

**Comment by juanruitina** [comment]
Please lowercase "computing" in "Confidential computing". Other than that LGTM! Feel free to change to UX+1 once you address that.

**Comment by camillet16** [comment]
@carkod Can we change the placement of the image based on the mockup?

Screenshot 2024-04-30 at 11 00 33 AM

There were also text differences between the mockup and the demo. If it's a copy doc situation then ignore this section
1. This was not in the mockup.
Screenshot 2024-04-30 at 11 02 28 AM
2. There's extra bullet points on the demo.
Screenshot 2024-04-30 at 11 02 18 AM

File: static/sass/styles.scss
Comment (lyubomir-popov) [inline static/sass/styles.scss:1303]: Just use the correct class = we have something in styles.scs called p-image-wrapper_2-3 or something like that - please don't hardcode.

**Comment by lyubomir-popov** [comment]
- [ ] `$nbsp;` between the last two words:

image


- [ ] we have p-hero for that now - no need to do the acrobatics with two nested p-strip>p-section

File: static/sass/styles.scss
Comment (mtruj013) [inline static/sass/styles.scss:1312]: Would you mind adding a comment linking this issue https://github.com/canonical/vanilla-framework/issues/4876? We should remember to remove once it finally makes it to Vanilla

File: templates/confidential-computing/ai.html
Comment (mtruj013) [inline templates/confidential-computing/ai.html:95]:
```suggestion
            <h3 class="p-heading--5">CPU-based confidential computing</h3>
```
This class doesn't seem to be doing anything here, is its usage intentional?

File: templates/confidential-computing/ai.html
Comment (mtruj013) [inline templates/confidential-computing/ai.html:110]:
```suggestion
        <hr class="p-rule is-muted" />
```

File: templates/confidential-computing/ai.html
Comment (mtruj013) [inline templates/confidential-computing/ai.html:116]:
```suggestion
            <h3 class="p-heading--5">GPU-based confidential computing</h3>
```

File: templates/confidential-computing/ai.html
Comment (mtruj013) [inline templates/confidential-computing/ai.html:131]:
```suggestion
        <hr class="p-rule is-muted" />
```

File: templates/confidential-computing/ai.html
Comment (mtruj013) [inline templates/confidential-computing/ai.html:148]:
```suggestion
            <hr class="p-rule is-muted" />
```

File: templates/confidential-computing/ai.html
Comment (mtruj013) [inline templates/confidential-computing/ai.html:207]:
```suggestion
        <p><a href="/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a></p>
```
Flagging since I've gotten this feedback from Lyubo, links should be wrapped in paragraphs

File: static/sass/styles.scss
Comment (carkod) [reply static/sass/styles.scss]: Sure, done. These 2 were added on the fly with Lyubo on a call.

File: templates/shared/forms/interactive/confidential-computing.html
Comment (mtruj013) [inline templates/shared/forms/interactive/confidential-computing.html:22]:
```suggestion
                <label for="customerName" class="is-required">Name:</label>
```

Comment (mtruj013) [reply templates/shared/forms/interactive/confidential-computing.html:22]: Please add to all required fields

File: templates/shared/forms/interactive/confidential-computing.html
Comment (mtruj013) [inline templates/shared/forms/interactive/confidential-computing.html:28]:
```suggestion
```

Comment (mtruj013) [reply templates/shared/forms/interactive/confidential-computing.html:28]: Please apply for all other inputs with this repeat attribue

**Review by mtruj013 — APPROVED** [review]
Thanks Carlos, a few comments in the code. Also, is the form supposed to have a textarea? There's one on the [doc](https://docs.google.com/document/d/1F4QYakPaIAL5fcttOnegkldTiWQ_tRon9apfbradX08/edit#heading=h.3x58yvung0in) but not sure that's out of date

**Pull request merged**
